python/test/unit/language/test_matmul.py

```diff
@@ -889,21 +889,26 @@ def test_mxfp8_mxfp4_matmul(M, N, K, BLOCK_M, BLOCK_N, BLOCK_K, NUM_STAGES, B_TR
         if (A_DATA_TYPE == 'float4' and not WITH_A_SCALE) or (B_DATA_TYPE == 'float4' and not WITH_B_SCALE):
             pytest.skip("Float4 without scale is tested in test_block_scale_fp4")
 
-    if B_DATA_TYPE != 'float4' and B_TRANS:
-        pytest.skip(f'No need to transpose B for {B_DATA_TYPE}')
-
     if not is_hip() and BLOCK_N == 256 and BLOCK_K == 256:
         NUM_STAGES = 2
 
     torch.manual_seed(42)
 
-    def create_operand(dtype: str, size0: int, size1: int, k_dim: int, transpose: bool = False):
+    def create_operand(dtype: str, size0: int, size1: int, k_dim: int, transpose: bool = True):
```

Comment:
Why changing the default for `transpose`? If to change it, we need to update call sites, which is not done.

Reply:
Before the change, A has T layout for fp8 and bf8 but has N layout for fp4 which misaligned with each other. This change is meant to corret that. Now A always has T layout for fp8, bf8 and fp4. And for B, addition to the original T and N layouts for fp4, we now support T and N layouts for fp8 and bf8.

Reply:
I see. This makes sense now; thanks for the explanation!